x-pack/plugins/infra/public/components/subscription_splash_content.tsx

```diff
@@ -103,16 +102,62 @@ export const SubscriptionSplashPrompt: React.FC = () => {
     );
   }
 
+  return (
+    <PageTemplate {...templateProps} isEmptyState={true}>
+      <EuiEmptyPrompt
+        iconType={'visLine'}
+        title={<h2>{title}</h2>}
+        body={
+          <EuiText>
+            <p>{description}</p>
+          </EuiText>
+        }
+        actions={cta}
```

Comment:
What does `cta` mean?

Reply:
I didn't make the changes here - I just moved them so we can have the page component unchanged and the old version of `SubscriptionSplashPrompt` is now part of the `SubscriptionSplashPage`. So the `cta` is the old code of the call to action button.

Reply:
maybe it means `call to action`